san/execute_sql.py

```diff
@@ -0,0 +1,85 @@
+import san.sanbase_graphql
+import pandas as pd
+from san.graphql import execute_gql, get_response_headers
+from san.query import get_gql_query, parse_dataset
+from san.transform import transform_timeseries_data_query_result
+from san.error import SanError
+import json
+
+def execute_sql(**kwargs):
+    """
+    Execute an arbitrary SQL against Santiment's Clickhouse Database
+
+    Example:
```

Comment:
Do we need this example when we have it in the `README.md` ?

Reply:
On the opposite, I think we should extend the documentation on the functions themselves. This is used when you type in the python shell `help(san.execute_sql)`, so you don't have to go to online doc tools.